lib/cc-reno.c

```diff
@@ -26,15 +26,19 @@
 #define QUICLY_MIN_CWND 2
 #define QUICLY_RENO_BETA 0.7
 
-void quicly_cc_init(quicly_cc_t *cc)
+void quicly_cc_init(quicly_cc_t *cc, uint32_t max_udp_payload_size)
 {
     memset(cc, 0, sizeof(quicly_cc_t));
-    cc->cwnd = QUICLY_INITIAL_WINDOW * QUICLY_MAX_PACKET_SIZE;
+    cc->cwnd = QUICLY_INITIAL_WINDOW * max_udp_payload_size;
+    if (cc->cwnd > 14720)
```

Comment:
Can you make this a QUICLY_MAX_INITIAL_WINDOW_BYTES config variable?

Reply:
With the changes made in 88e2ddf, aba039a, `quicly_cc_init` now accepts the CWND size as a parameter rather than trying to calculate by its own.

This opens us the possibility of tuning INITCWND in various ways in the future.

Now, the default logic for calculating the CWND is in `quicly_cc_calc_initial_cwnd`, and the function has constants being declared.

PTAL.